### ISSUE TITLE
No Dialogue

### ISSUE THREAD

**Issue by DarkRaider12** [Ladysnake/Requiem#661]
when i die i cant interact with anything and theirs no dialogue so i have to force kill myself with cheats which i wont really like doing. So is there someway to fix this compact issue?

**Comment by Redfan2** [comment]
Could you send a game log?

**Comment by DarkRaider12** [comment]
Once I get home

**Comment by DarkRaider12** [comment]
> Could you send a game log?

[Uploading latest.log…]()

**Comment by DarkRaider12** [comment]
[latest.log](https://github.com/user-attachments/files/15779496/latest.log)

**Comment by DarkRaider12** [comment]
redfan2?

**Comment by DarkRaider12** [comment]
This is a better log to look at 
[2024-06-12-4.txt](https://github.com/user-attachments/files/15828519/2024-06-12-4.txt)

**Comment by Redfan2** [comment]
Does this look like in #634?

**Comment by DarkRaider12** [comment]
> Does this look like in #634?

yes but when i put origin on my server whne i put demon and human it instantly makes my a remnant

**Comment by DarkRaider12** [comment]
im having the same issue with #639

**Comment by DarkRaider12** [comment]
it doesn't do the same in singleplayer

**Comment by DarkRaider12** [comment]
[Uploading latest.log…]()

**Comment by DarkRaider12** [comment]
[Uploading latest.log…]()

**Comment by DarkRaider12** [comment]
https://mclo.gs/BG0T7p1
